src/uikit.cs

```diff
@@ -10246,6 +10277,28 @@ public interface UITabBarItem : NSCoding {
 		[Since (7,0)]
 		[Export ("selectedImage", ArgumentSemantic.Retain)][NullAllowed]
 		UIImage SelectedImage { get; set; }
+
+		[iOS (10, 0)]
+		[NullAllowed, Export ("badgeColor", ArgumentSemantic.Copy)]
+		UIColor BadgeColor { get; set; }
+
+		[iOS (10,0)]
+		[Export ("setBadgeTextAttributes:forState:")]
+		[Internal]
+		void SetBadgeTextAttributes ([NullAllowed] NSDictionary textAttributes, UIControlState state);
+
+		[iOS (10,0)]
+		[Wrap ("SetBadgeTextAttributes (textAttributes.Dictionary, state)")]
```

Comment:
This will cause an NRE if `null` is used, which is valid since the API it calls has `[NullAllowed]`